src/components/pages/system/profile/ProfileEdit.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useFormik } from 'formik';
+import Yup from 'utils/yup';
+
+import Button from 'components/form-components/Button';
+import Form from 'components/form-components/Form';
+import TextInput from 'components/form-components/TextInput';
+import { setUserData } from 'store/user/actionCreators';
+
+const ProfileEdit = () => {
+  const user = useSelector((store) => store.user);
+  const dispatch = useDispatch();
+
+  const formik = useFormik({
+    initialValues: user,
+    validationSchema: Yup.object({
+      first_name: Yup.string().required(),
+      last_name: Yup.string().required(),
+      company_name: Yup.string(),
+      email: Yup.string().required().email(),
+      instagram: Yup.string(),
+      twitter: Yup.string(),
+    }),
+    onSubmit: (values, actions) => {
+      // todo: post data to the backend and save to state from response
```

Comment:
use caps for todo annotations
```suggestion
      // TODO: post data to the backend and save to state from response
```